config/locales/core.bs.yml

```diff
@@ -96,4 +96,4 @@ bs:
     pagination:
       next_title: Veza ka sljedećoj stranici
       previous_title: Veza ka prethodnoj stranici
-      title: 'Redni broj stranice: '
\ No newline at end of file
+      title: 'Redni broj stranice: '
```

Comment:
Review the line break here. As the only change in this file should be the line 88